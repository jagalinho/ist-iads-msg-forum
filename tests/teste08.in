A 634 43SQN3NCSEC1X29IIV0BC1TAU6XKZEUSCXKHLHZ
T
A 24 U6GTMYAE4A1U0MJIZPOA6CFHPC3J4O4Q231N9BCBFCNCUTUFFXBFP7EJF6ZLITN2NF9IGWC39N
A 24 XCKTDHX5P6AOFF7RRI
T
A 45 5DLA43CZGZWFJQXVONQGRGDTRM893NVFU0F681ZCYOBEJ
A 45 UN6CRXL1FBBQ0EFUOSBYD9BXKMB2LUN8S20U9PIP71RL5ACVGUV0DVD2381K334V2V26TY9ZZTDOOT6Z0FPK50
A 45 W4Y4A3OF36WOSWBVWC6WQ61448TRG9J7NUNQJMUFY8J2Z
T
A 924 LPSCGAIZFJKOWM3TRIZWQNU6ZWCLL7G9ONP1NXB3R5GP3FSFEXXQMC9INIRZHT63FRIO76ODWFPIGAQ4M1IU4SOK4A4EUNKWJCX5HF15H4UOWC7SD69YFX12Y9XV8AKY
A 924 D9SBBXSFF4LJ25MMIAV1EQ10Y8KV44SY9B9VKU54KBWPJTGU79NMDSF31VUP728O0AEC9C95VXV0R8X0NW1U220LWM406QFYK5WSKRI9FBVXGXW8RL5WVBB
A 924 GWR1CLMO6QE9URLBFWKGNXZJ8EX7
A 924 RBJ4B5T2FR979TPO0Y79N67YB3QWX06N7KF2B
T
A 638 X1VT65E5Q2LH9XER0GXYZBYUXZY7NZ27LMRKB2TBOXNUUXM4BGUL3SYX08YLSHG105NXYIOS35OP2WHFDH4R8IUF4Z8NDZCD4ZEIZ32KC5PZJVG83A9E1YVEYBQ1
A 638 11GJJTABK4FW77XD8C7HTP0SA4B9WVXUNF5E5C19SRNSXFSEEEIBB8R41SE4F94IITW6GZ
A 638 RER6DU490046TRRLT0JI0
A 638 HVB580DLA27CVV2H4ZT338SOW9SPX3O6SOEUG28VKRAK60ILUDSCOW476Y1IR7X1KJTJEAX7GW398FP80MDL0ZR2SFAAI8UVP151I80LP34
A 638 MY7D7WKE9CCUFSRGIVVNYXU22BAPD50UNMRBU1A5RG2VZ4NWF9OFG4Z37VZF77ELHQL5H9LEOCT1VUFJ7P9YZG0NPBCTEUEZ93N6WHZGCAMZ0IGAX2GUVKGFIR8GHW
T
A 279 Z37PQC2E3PIR8APTATS6VBELNWIVJWQNONFTJQ5QFI47Y52R08UO2BL5HL1A6V1NNYEX64P9YUJ8OGWE5SE0IKLOME2N1S0UPOILTUJZA03D30384Z4SALM7S8LYYVG
A 279 XPMXMYWC4CL3JRNM14DAKIZ9Q1IO9MTCQEMTN0WA6NLSH6FEE4RPPENKTX8675HHQUZBG2P8SFBUIYIGWIB0
A 279 CKR1Q9P037PWMLWM6H7FRDI5BWCUDOHBHDLE5TKCBUT14OJNE8L972VR18GJ8ALTE3YZQEJO2Z151C41YCOCFT1GZWXQQNDC2MLZUMP5LLSWUI1
A 279 WQUH2KGRYWNYE3SAX4XE0XX3760QEZX5XC3LGIXD3NI677KW39I70L9WUQULDPT0IIOH3JE3Z3X38ZVVOELBH1M88YWF3BTPKECOFCIXI4AFOG627JH3ZSXY2
A 279 FCTDRRQ6F0PBZ8HG1TXY
A 279 NHI8LUU
T
X
