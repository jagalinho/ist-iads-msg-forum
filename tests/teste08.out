*MOST ACTIVE USER:634:1
*MOST ACTIVE USER:24:2
*MOST ACTIVE USER:45:3
*MOST ACTIVE USER:924:4
*MOST ACTIVE USER:638:5
*MOST ACTIVE USER:279:6
21
